app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -239,83 +243,103 @@ fun FilterSection(
     onToggle: () -> Unit,
     content: @Composable () -> Unit
 ) {
-  Column {
-    Text(
-        text = title,
-        style = MaterialTheme.typography.titleMedium,
-        modifier =
-            Modifier.clickable(onClick = onToggle)
-                .padding(8.dp)
-                .background(Color.LightGray)
-                .fillMaxWidth(),
-        color = MaterialTheme.colorScheme.tertiary,
-        testTag = title)
+  Column(
+      modifier =
+          Modifier.padding(8.dp)
+              .border(1.dp, Color.Gray, shape = MaterialTheme.shapes.medium)
+              .background(MaterialTheme.colorScheme.surface, shape = MaterialTheme.shapes.medium)
+              .padding(8.dp)) {
+        Text(
+            text = title,
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.clickable(onClick = onToggle).padding(8.dp).fillMaxWidth(),
+            color = MaterialTheme.colorScheme.primary,
+            testTag = title)
 
-    if (isExpanded) {
-      content()
-    }
-  }
+        if (isExpanded) {
+          content()
+        }
+      }
 }
 
+@SuppressLint("DefaultLocale")
```

Comment:
I'm not sure that explicitly removing a warning is a good thing, moreover when Android Studio is telling you "*Implicitly using the default locale is a common source of bugs: Use `String.format(Locale, ...)` instead*".

Reply:
Sometimes you kind of have to suppress warnings, but in this case since Android studio explicitly suggests an alternative it might be good to use it just in case